data/scripts/suite.py

```diff
@@ -139,9 +140,21 @@ def run_suite(args):
     parser.add_argument("--valgrind", help="Path to valgrind, if provided " \
                         "the tests are run with it", type=str)
     parser.add_argument("--valgrind-supp", help="Path to valgrind's suppression file", type=str)
+    parser.add_argument("--color", dest="color", help="Disable colored output", action="store_true")
+    parser.add_argument("--no-color", dest="color", help="Force enable colored output", action="store_false")
```

Comment:
Can't this be a `--color=true`, `--color=auto`, or `--color=false` instead of two options?